man/sd_bus_add_match.xml

```diff
@@ -153,6 +153,15 @@
 
   <xi:include href="libsystemd-pkgconfig.xml" />
 
+  <refsect1>
+    <title>History</title>
+    <para><function>sd_bus_add_match()</function> was added in version 231.</para>
+    <para><function>sd_bus_message_handler_t()</function> was added in version 231.</para>
```

Comment:
All the callback function types (suffixed by `_t`) shouldn't be annotated as functions. Maybe they shouldn't be included in the changelog at all? Not sure on this one.

Reply:
Makes sense. As you can guess, this was done mechanically and sd_bus_message_handler_t was declared with a [funcprototype](https://github.com/systemd/systemd/blob/6d057784691139aa2d9964d76f5c6de2a8f5e0be/man/sd_bus_add_match.xml#L36)